tests/lora/test_lora_layers_peft.py

```diff
@@ -1727,6 +1729,20 @@ def test_load_unload_load_kohya_lora(self):
         self.assertTrue(np.allclose(lora_images, lora_images_again, atol=1e-3))
         release_memory(pipe)
 
+    def test_empty_state_dict(self):
```

Comment:
Thank you but can we maybe add this as a fast test with smaller ckpts?

Reply:
I think having two slow tests is fine